dev/inc/RoutedEventHelpers.h

```diff
@@ -28,10 +28,10 @@ struct RoutedEventHandler_revoker
         return *this;
     }
 
-    RoutedEventHandler_revoker(winrt::UIElement const& object, winrt::RoutedEvent const& event, winrt::IInspectable const& handler) :
+    RoutedEventHandler_revoker(winrt::UIElement const& object, winrt::RoutedEvent  event, winrt::IInspectable  handler) :
         m_object(object),
-        m_event(event),
-        m_handler(handler)
+        m_event(std::move(event)),
+        m_handler(std::move(handler))
```

Comment:
what is the rule for when std::move should be added here? (does clang have this written down?)

Reply:
The full list is here. I don't think there is just one rule for move. It might be spread across a few checks.
http://clang.llvm.org/extra/clang-tidy/checks/list.html

Reply:
Does clang tell you when it's tidying which rules it's applying on which lines?

Reply:
Yes. In verbose logs it does. For this change I believe it is this check that caused it to change.
https://clang.llvm.org/extra/clang-tidy/checks/performance-unnecessary-value-param.html

Reply:
Ok and it looks like it made that change because it also changed const& to by-value. Presumably if it's const& param then it doesn't std::move?